web/src/components/Shape.js

```diff
@@ -70,6 +70,7 @@ export class Shape extends Component {
                         <option value="box">Box</option>
                     </select>
                     </label>
+                    {/* CR * seeing this code again */}
```

Comment:
Looking pretty WET over here! 🌊 You should DRY this up with some centralized components.